### PR TITLE
♻️ Update twilio

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

**Comment by rowanseymour** [comment]
@norkans7 this tested on staging yet?

File: temba/channels/types/twilio/views.py [hunk 1]
```diff
@@ -131,13 +131,13 @@ def claim_number(self, user, phone_number, country, role):
 
         is_short_code = len(phone_number) <= 6
         if is_short_code:
-            short_codes = client.sms.short_codes.list(short_code=phone_number)
+            short_codes = list(client.api.short_codes.stream(short_code=phone_number))
```
Comment (dodobas) [inline temba/channels/types/twilio/views.py]: this is just something to think about, 

`.stream` looks like an generator, and that's why we are caching the results by using `list`, in the next few lines we only really care about the first short_code and in that case we could do something like:
```
short_codes_api_response = client.api.short_codes.stream(short_code=phone_number)

short_code = next(short_codes_api_response, None)

if short_code:
...
```

File: temba/channels/views.py [hunk 2]
```diff
@@ -1800,13 +1800,13 @@ def search_available_numbers(self, client, **kwargs):
 
             kwargs["type"] = "local"
             try:
-                available_numbers += client.phone_numbers.search(**kwargs)
+                available_numbers += client.api.available_phone_numbers.stream(**kwargs)
```
Comment (dodobas) [inline temba/channels/views.py]: is `stream` is a generator this will raise a TypeError exception

File: temba/tests/twilio.py [hunk 3]
```diff
@@ -15,81 +15,137 @@ class MockTwilioClient(TwilioClient):
...
-    class MockShortCodes(object):
+    class MockShortCodes(MockInstanceResource):
         def __init__(self, *args):
             pass
 
         def list(self, short_code=None):
             return [MockTwilioClient.MockShortCode(short_code)]
 
+        def stream(self, *args, **kwargs):
+            return [MockTwilioClient.MockShortCode("1122")]
```
Comment (dodobas) [inline temba/tests/twilio.py]: are we sure that actual API call returns a list ? 

maybe it should be a generator

**Comment by rowanseymour** [comment]
@norkans7 what exactly are we waiting on there? Another Twilio release?

**Pull request merged**